### DISCUSSION TITLE
the simulation does not start

### DISCUSSION THREAD

**Discussion by ghaimaroua** [firemodels/fds#11627] [General]
hi im student and it is my first time in using fds i have some issue please help me 
i use FDS6.7.9 on windows 11 
i use Pyrosim and fds  for the simulation and i got this message and the simulation does not work 

C:\Users\ghais\Desktop\please>fds_local -p 1 -o 16 last.fds

 Starting FDS ...

 MPI Process      0 started on Administrator

 Reading FDS input file ...

WARNING: SPEC REAC_FUEL is not in the table of pre-defined species. Any unassigned SPEC variables in the input were assigned the properties of nitrogen.

 Fire Dynamics Simulator

 Current Date     : April  3, 2023  01:14:13
 Revision         : FDS6.7.9-0-gec52dee42-release
 Revision Date    : Sun Jun 26 14:36:40 2022 -0400
 Compiler         :
 Compilation Date : Tue 06/28/2022  11:17 PM

 MPI Enabled;    Number of MPI Processes:       1
 OpenMP Enabled; Number of OpenMP Threads:     16

 MPI version: 3.1
 MPI library version: Intel(R) MPI Library 2021.6 for Windows* OS

 Job TITLE        :
 Job ID string    : last
thank you so much

**Comment by rmcdermo** [comment]
You have not demonstrated an error.  The warning just says you are using a fuel that is not in the list of pre-defined.  It is ok, it will just not have the thermophysical properties of the actual material.

This should be posted to the Discussions page.  I think there should be a way to bounce it over there... I'll try.

**Comment by rmcdermo** [comment]
Ok, that was easy... there is a button on the right side (for maintainers).

If you need to continue the discussion, we will need an input file (not a Pyrosim file, but the .fds file; and you will need to change the extension to .txt so it can be uploaded to GitHub.

**Comment by ghaimaroua** [comment]
do you mean i should change the type of matrial used in the project 
here is the file of the fds model (i deletede the copyed sentences of the walls so the comment is not too long) 

maroua.fds
Generated by PyroSim - Version 2022.3.1208
3 avr. 2023 00:06:58

&HEAD CHID='maroua'/
&TIME T_END=300.0/
&DUMP DT_PL3D=10.0, DT_SL3D=0.25, WRITE_XYZ=.TRUE., PLOT3D_QUANTITY(1)='VISIBILITY'/

&MESH ID='Maillage01-01-01', IJK=372,187,55, XB=0.0,74.4,0.0,37.4,0.0,11.0/
&MESH ID='Maillage01-01-02', IJK=393,187,55, XB=74.4,153.0,0.0,37.4,0.0,11.0/
&MESH ID='Maillage01-02-01', IJK=372,208,55, XB=0.0,74.4,37.4,79.0,0.0,11.0/
&MESH ID='Maillage01-02-02', IJK=393,208,55, XB=74.4,153.0,37.4,79.0,0.0,11.0/


&REAC ID='SFPE POLYURETHANE_GM27',
      FYI='SFPE Handbook, 5th Edition, Tables A.38 and A.39',
      FUEL='REAC_FUEL',
      C=1.0,
      H=1.7,
      O=0.3,
      N=0.08,
      SOOT_YIELD=0.05,
      HEAT_OF_COMBUSTION=1.64E+4,
      RADIATIVE_FRACTION=0.537/

&MATL ID='CONCRETE',
      FYI='NBSIR 88-3752 - ATF NIST Multi-Floor Validation',
      SPECIFIC_HEAT=1.04,
      CONDUCTIVITY=1.8,
      DENSITY=2280.0/

&SURF ID='Ceiling Tile 600 x 1200',
      RGB=179,178,175/
&SURF ID='Default Roof',
      RGB=80,80,80/
&SURF ID='Gypsum Wall Board',
      RGB=171,170,168/
&SURF ID='Default Wall',
      RGB=80,80,80/
&SURF ID='993kw fire',
      COLOR='RED',
      HRRPUA=100.0,
      TAU_Q=-200.0,
      TMP_FRONT=300.0/
&SURF ID='Concrete',
      RGB=146,202,166,
      BACKING='VOID',
      MATL_ID(1,1)='CONCRETE',
      MATL_MASS_FRACTION(1,1)=1.0,
      THICKNESS(1)=0.1/

&OBST ID='Compound Ceiling:600 x 1200mm grid:226475', XB=3.4,74.4,18.8,22.2,5.6,5.6, RGB=179,178,175, SURF_ID='Ceiling Tile 600 x 1200'/ 
&OBST ID='Compound Ceiling:600 x 1200mm grid:226475', XB=3.6,43.8,18.2,18.4,5.6,5.6, RGB=179,178,175, SURF_ID='Ceiling Tile 600 x 1200'/ 
&OBST ID='Compound Ceiling:600 x 1200mm grid:226475', XB=3.6,74.4,18.4,18.8,5.6,5.6, RGB=179,178,175, SURF_ID='Ceiling Tile 600 x 1200'/ 
&OBST ID='Compound Ceiling:600 x 1200mm grid:226475', XB=3.6,74.4,22.2,23.8,5.6,5.6, RGB=179,178,175, SURF_ID='Ceiling Tile 600 x 1200'/ 
&OBST ID='Compound Ceiling:600 x 1200mm grid:226475', XB=3.8,39.4,17.6,17.8,5.6,5.6, RGB=179,178,175, SURF_ID='Ceiling Tile 600 x 1200'/ 
&OBST ID='Compound Ceiling:600 x 1200mm grid:226475', XB=3.8,40.4,17.8,18.0,5.6,5.6, RGB=179,178,175, SURF_ID='Ceiling Tile 600 x 1200'/ 
&OBST ID='Compound Ceiling:600 x 1200mm grid:226475', XB=3.8,41.6,18.0,18.2,5.6,5.6, RGB=179,178,175, SURF_ID='Ceiling Tile 600 x 1200'/ 
&OBST ID='Compound Ceiling:600 x 1200mm grid:226475', XB=3.8,71.0,24.8,25.0,5.6,5.6, RGB=179,178,175, SURF_ID='Ceiling Tile 600 x 1200'/ 
&OBST ID='Compound Ceiling:600 x 1200mm grid:226475', XB=3.8,71.6,24.6,24.8,5.6,5.6, RGB=179,178,175, SURF_ID='Ceiling Tile 600 x 1200'/ 
&OBST ID='Compound Ceiling:600 x 1200mm grid:226475', XB=3.8,72.2,24.4,24.6,5.6,5.6, RGB=179,178,175, SURF_ID='Ceiling Tile 600 x 1200'/ 
&OBST ID='Compound Ceiling:600 x 1200mm grid:226475', XB=3.8,72.8,24.2,24.4,5.6,5.6, RGB=179,178,175, SURF_ID='Ceiling Tile 600 x 1200'/ 

&OBST ID='Floor:Generic Floor - 400mm:214916', XB=10.8,22.4,13.0,13.2,0.0,0.4, RGB=80,80,80, SURF_ID='Default Roof'/ 
&OBST ID='Floor:Generic Floor - 400mm:214916', XB=11.8,21.0,12.8,13.0,0.0,0.4, RGB=80,80,80, SURF_ID='Default Roof'/ 
&OBST ID='Floor:Generic Floor - 400mm:214916', XB=13.2,19.0,12.6,12.8,0.0,0.4, RGB=80,80,80, SURF_ID='Default Roof'/ 
&OBST ID='Floor:Generic Floor - 400mm:214916', XB=20.4,64.6,29.4,29.6,0.0,0.4, RGB=80,80,80, SURF_ID='Default Roof'/ 

&OBST ID='Basic Wall:Generic - 300mm:209890', XB=0.8,1.0,57.2,57.4,0.4,10.4, COLOR='WHITE', SURF_ID='Concrete'/ 
&OBST ID='Basic Wall:Generic - 300mm:209890', XB=0.8,1.2,57.4,57.6,0.4,10.4, COLOR='WHITE', SURF_ID='Concrete'/ 
&OBST ID='Basic Wall:Generic - 300mm:209890', XB=1.0,1.2,57.6,57.8,0.4,10.4, COLOR='WHITE', SURF_ID='Concrete'/ 
&OBST ID='Basic Wall:Generic - 300mm:209890', XB=1.0,1.4,57.8,58.0,0.4,10.4, COLOR='WHITE', SURF_ID='Concrete'/ 
&OBST ID='Basic Wall:Generic - 300mm:209890', XB=1.2,1.4,58.0,58.2,0.4,10.4, COLOR='WHITE', SURF_ID='Concrete'/ 
&OBST ID='Basic Wall:Generic - 300mm:209890', XB=1.2,1.6,58.2,58.4,0.4,10.4, COLOR='WHITE', SURF_ID='Concrete'/ 

&OBST ID='Basic Wall:Generic - 300mm:242606', XB=116.0,116.4,15.8,16.0,5.4,5.8, RGB=80,80,80, SURF_ID='Default Wall'/ 
&OBST ID='Basic Wall:Generic - 300mm:242606', XB=116.2,116.6,16.0,16.2,5.4,5.8, RGB=80,80,80, SURF_ID='Default Wall'/ 


&VENT ID='Ouverture maillage (Mesh Vent): Maillage01-01-01 [YMIN]', SURF_ID='OPEN', XB=0.0,74.4,0.0,0.0,0.0,11.0/ 
&VENT ID='Ouverture maillage (Mesh Vent): Maillage01-01-01 [ZMAX]', SURF_ID='OPEN', XB=0.0,74.4,0.0,37.4,11.0,11.0/ 
&VENT ID='Fire', SURF_ID='993kw fire', XB=40.150694,43.739355,66.040051,68.808447,0.4,0.4/ 

&BNDF QUANTITY='WALL TEMPERATURE'/

&ISOF QUANTITY='VISIBILITY', VALUE=10.0/

&SLCF QUANTITY='VISIBILITY', ID='A', PBX=42.527853/
&SLCF QUANTITY='VELOCITY', VECTOR=.TRUE., ID='B', PBY=63.389342/
&SLCF QUANTITY='VISIBILITY', ID='C', PBZ=1.309307/


&TAIL /
thank you

**Comment by mcgratta** [comment]
The WARNING message is telling you that your FUEL is understood to be a gas with the properties of nitrogen except for the molecular weight, which will be calculated from the formula.

**Comment by mcgratta** [comment]
One reason the simulation "does not start" is that each mesh has 4.5 million cells. That will take a long time run, assuming your computer has enough RAM. Consider using more meshes and splitting the job across several computers.

**Reply by ghaimaroua** [reply]
i think it is because i convert the project from the Revit in DXF format to Pyrosim the walls have been split into multiple walls i will try to draw directly in Pyrosim  
thank you for your help

**Reply by drjfloyd** [reply]
You choose the number of cells when you define the MESH in Pyrosim.

**Reply by ghaimaroua** [reply]
can you please show me how to define the number of cells

**Comment by drjfloyd** [comment]
This is the  issue tracker for the open source project for FDS. This is not a forum for the commerical product Pyrosim. First read the Pyrosim User's Guide and if you still have questions go to the Thunderhead Engineering webpage and use the Pyrosim resources there.

**Reply by ghaimaroua** [reply]
thank you i tried another small example and it worked, thank you so much for your help i appreciate it .
